cloud/endagaweb/templates/dashboard/tower_detail/deregister.html

```diff
@@ -98,25 +121,50 @@ <h4 class='modal-title'>
       </a>
     </p>
   </div>
+<!-- removed  check of Tower  status == 'No Data' -->
+  {% if endaga_version > '00000.00003.00019' %}
```

Comment:
Added check with proper value of tower status No Data having commit id 8f410e35e7aa6fbd7fcb79945516f9cd62e859a9